### DISCUSSION TITLE
Implement Peek - Preview Websites with ease

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#790] [Ideas]
### What feature would you like to see?

One of the best features of Arc is "Peek".

![opening-a-Peek-tab-in-Arc-browser](https://github.com/user-attachments/assets/5327baa1-1f35-4611-b096-caa6e6883f42)

---

You can find it explained in detail here:
https://resources.arc.net/hc/en-us/articles/19335302900887-Peek-Preview-Sites-From-Pinned-Tabs

It would be amazing if Zen also implemented it, and with the performance improvements that this browser brings, this feature could be immensely useful.

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/592_

**Comment by gustavomarchesim** [comment]
I would love this functionality

**Comment by amjd** [comment]
Would be great to have this natively in Zen. Until then, I'm using this addon to replicate that functionality:
https://addons.mozilla.org/en-US/firefox/addon/maxfocus-link-preview/

It uses iframe, so not all websites load in the preview, but those can be opened in a new tab by clicking the link.

**Comment by eden881** [comment]
I think this can be closed? idk when it was implemented but it works in `1.7b`

**Comment by Taaqif** [comment]
Zen has this natively, its called Glance.
Enable it in `Settings -> Look and Feel -> Glance`
You can change your preferred shortcut to trigger it
